_1327/static/less/1327.less

```diff
@@ -226,3 +226,17 @@ footer a {
 table.table-narrow {
 	max-width: 400px;
 }
+
+.has-error .md-editor {
+  .form-control-validation(@state-danger-text; @state-danger-text; @state-danger-bg);
```

Comment:
What does this do? I can't tell the difference between using this and commenting it out.